pages/curriculum/[lessonSlug]/mentor/addExercise/index.tsx

```diff
@@ -43,10 +44,19 @@ const Header = <
   setModule,
   setErrorMsg
 }: HeaderProps<T>) => {
+  const { query, push } = useRouter()
+  const { lessonSlug } = query
+
   const modules = get(lesson, 'modules') ?? []
 
   return (
     <header className={styles.header}>
+      <button
+        className="btn ps-0 d-flex align-items-center mb-3"
+        onClick={() => push(`/curriculum/${lessonSlug}/mentor`)}
```

Comment:
Another way to do this would be to use a link so you don't need to bother with onClick and router push:
```tsx
<Link href={`/curriculum/${lessonSlug}/mentor`}>
  <a><ArrowLeftIcon size="medium" aria-label="Exit" /></a>
</Link>
```

Reply:
This one is shorter and is handled by next.js. I'll update the code.

Reply:
Done.